extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -69,7 +70,7 @@ class AnalyticsEvent {
 
   /**
    * @param {!AnalyticsEventType} type The type of event.
-   * @param {!Object<string, string>} A map of vars and their values.
+   * @param {!Object<string, string>} [vars] A map of vars and their values.
```

Comment:
It is optional argument. You dont allocate optional args by `[paramName]`?
http://usejsdoc.org/tags-param.html